crates/cairo-annotations/Cargo.toml

```diff
@@ -27,3 +27,8 @@ regex.workspace = true
 [dev-dependencies]
 assert_fs.workspace = true
 snapbox.workspace = true
+
+[features]
+default = ["cairo-lang"]
```

Comment:
I would not enable this by default - if someone needs this, they can opt in.

Reply:
I did this for backwards compatibility, my thought is that maybe the default behaviour should not be changed 🤔

Reply:
+ scarb is actually the only thing that does NOT need this on

Reply:
yes, but why :D if someone does not need this, they can opt out (and most need this actually)

Reply:
Fair enough, changed :+1: